db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
```

Comment:
we have `jsonb` in our migrations, e.g. [showback](https://github.com/ManageIQ/manageiq-schema/blob/27582ecff95dfcb740a8e3d22d2e7c3e03ea46fe/db/migrate/20170519153350_add_showback_models.rb#L35) lately

maybe look at their ActiveRecord model about serializing. I have never looked at how we do it